src/cijoe/core/command.py

```diff
@@ -150,19 +150,20 @@ def __init__(self, config: Config, output_path: Path, monitor: bool):
 
         os.makedirs(os.path.join(self.output_path, self.output_ident), exist_ok=True)
 
-        self.transport_local = transport.Local(self.config, self.output_path)
         self.transports = {}
 
         config_transports = self.config.options.get("cijoe", {}).get("transport", {})
         for transport_name in config_transports:
+            if transport_name == "initiator":
+                log.warning(
+                    "The transport name 'initiator' is reserved. The defined transport with this name will be skipped."
```

Comment:
Let's make this a hard error.

Someone manually crafting a configuration file for a system with a similar initiator/target concept might overlook the warning, leading to unexpected behavior.

Reply:
We have agreed to remove it, because it's also being caught in the constructor of the Config object - and this is always initialised when calling the Cijoe constructor.